### PR TITLE
Apply jshint rules to various places in UI app.

### PR DESCRIPTION
**Pull request by anwar6953**
This is a lot of semicolons, but also a few other lines of cleanup. Helped find a few issues.

Note: not all JSHint warnings are resolved in this PR.

### CODE REVIEW

File: cdap-ui/app/services/helpers.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ angular.module(PKG.name+'.services')
         var finalKey = timeValues[roundedDown];
         aggregated[finalKey] = 0;
         for (var i = roundedDown; i < timeValues.length; i++) {
-          aggregated[finalKey] += inputMetrics[timeValues[j]]
```
Comment (anwar6953) [inline cdap-ui/app/services/helpers.js]: Was using the wrong iteration variable, `j` instead of `i`.

File: cdap-ui/app/services/namespace.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ angular.module(PKG.name + '.services')
       var ns = this.namespaceList.filter(function(namespace) {
         return namespace.name === name;
       });
-      return ns[0].name || id;
+      return ns[0].name || name;
```
Comment (anwar6953) [inline cdap-ui/app/services/namespace.js]: And this.

File: cdap-ui/app/features/adapters/controllers/create-ctrl.js [hunk 3]
```diff
@@ -231,7 +233,7 @@ angular.module(PKG.name + '.feature.adapters')
           delete sink.properties[key];
         }
       });
-      for (var i=0; i<trans.length; i++) {
+      for (i=0; i<trans.length; i++) {
```
Comment (bdmogal) [inline cdap-ui/app/features/adapters/controllers/create-ctrl.js]: spaces?

Comment (anwar6953) [inline cdap-ui/app/features/adapters/controllers/create-ctrl.js]: Spacing is not consistent all over the place.
I'm going to break it down by category, and just apply a certain set of changes per PR.
So, not everything will go in this PR.

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Pull request merged**